### ISSUE TITLE
[enhancement] Add import alias option to cli tools

### ISSUE THREAD

**Issue by edwardsCam** [format-message/format-message#232]
_tl;dr Add an option that allows me to customize the import name, instead of assuming it's 'format-message'._

---

From format-message-cli readme:
> All of the command line tools will look for requireing or importing format-message in your source files to determine the local name of the formatMessage function.

It would be nice if this were configurable (but still defaults to 'format-message' if you don't use it).

In my case, I'm exporting a package that acts as a wrapper to format-message. It's basically a namespace of format-message, plus some additions. My code imports this wrapper instead of format-message directly. However, it has a different name, so when I `format-message extract` it does not pick up any strings:

```javascript
import t from '@inst/bridge-ui-components.format-message'
...
t('This string never gets extracted! :( ')
```

It would be great if I could tell the static analyzer to look for `@inst/bridge-ui-components.format-message` instead of `format-message`

This would also be useful if you're using a webpack alias. If, say, your code imports from a different name than `format-message`, the cli right now won't work as expected:

```javascript
// webpack.config.js
module.exports = {
  resolve: {
    alias: {
      'translation': 'format-message'
    }
  }
};
```
```javascript
// webpack aliases translation to format-message
import t from 'translation'
```

This could default to searching for `'format-message'` as it does already, so this would be a non-required option and would not break existing customers.

## Proposed usage
`format-message extract --alias <NAME_TO_SEARCH_FOR> src/**/*.js`
`format-message extract -a <NAME_TO_SEARCH_FOR> src/**/*.js`

_(open to suggestions on a different name, `-a` or `--alias` makes sense to me though)_

**Comment by vanwagonet** [comment]
The current functionality looks for `format-message` as the last segment of the import specifier or require path, so `require("../lib/format-message")` or `import _ from "@lib/i18n/format-message"` etc should work. If there are still cases where this is not feasible, I think adding this feature is reasonable, if you'd like to submit a pull request.

**Comment by jimsimon** [comment]
We're running into this issue as well.  Being able to override the regex used in babel-plugin-extract-format-message would be a welcome feature.

**Comment by jimsimon** [comment]
To clarify, one of the spots that this breaks down is when centralizing format-message configuration in a monorepo:
```
import formatMessage from 'format-message'

formatMessage.setup(...)

export { formatMessage }
```
and then...
`import { formatMessage } from '@instructure/quiz-format-message'`

**Comment by raybrownco** [comment]
@vanwagonet We can close this issue now that 6.2.4 is out
